grpc.go

```diff
@@ -1571,6 +1571,10 @@ func (a *agentGRPC) ListRoutes(ctx context.Context, req *pb.ListRoutesRequest) (
 	return a.sandbox.listRoutes(nil)
 }
 
+func (a *agentGRPC) AddARPNeighbors(ctx context.Context, req *pb.AddARPNeighborsRequest) (*gpb.Empty, error) {
```

Comment:
This feature will also need to be added to the new rust agent.

/cc @lifupan.

Reply:
@jodh-intel 
 I had opened an issue to track it on rust-agent repo:  https://github.com/kata-containers/kata-containers/issues/218

Reply:
Thanks @lifupan !